.configure

```diff
@@ -5,31 +5,31 @@
   "files_to_copy": [
     {
       "file": "iOS/simplenote/SPCredentials.swift",
-      "destination": "Simplenote/Credentials/SPCredentials.swift",
+      "destination": "~/.configure/simplenote-ios/secrets/SPCredentials.swift",
```

Comment:
As a future improvement, I think we could make `configure` default to `~/.configure/<"project_name" key value>/secrets` if no `destination` is specified.

One reason _not_ to do it is that, while it would keep the config file DRY and avoid the chance of typos, it would make it less clear to newcomers where the files go. ⚖️

Reply:
I like the idea of doing this, but making the logging more verbose so it's clear what the tool is doing under the hood?

Reply:
Great point!